Content.Shared/_DV/Holosign/ChargeHolosignSystem.cs

```diff
@@ -36,24 +36,13 @@ private void OnInit(Entity<ChargeHolosignProjectorComponent> ent, ref ComponentI
         if (string.IsNullOrEmpty(ent.Comp.SignComponentName))
             return;
 
-        ent.Comp.Container = _container.EnsureContainer<Container>(ent, ent.Comp.ContainerId);
         ent.Comp.SignComponent = EntityManager.ComponentFactory.GetRegistration(ent.Comp.SignComponentName).Type;
     }
 
     private void OnMapInit(Entity<ChargeHolosignProjectorComponent> ent, ref MapInitEvent args)
     {
         if (!TryComp<LimitedChargesComponent>(ent, out var charges))
             return;
-
```

Comment:
Looks like you could just remove this whole function now, and its constructor subscription

Reply:
Done.